pyfstat/helper_functions.py

```diff
@@ -57,39 +56,13 @@ def set_up_command_line_arguments():
         action="store_true",
         help="Decrease output verbosity [logging.WARNING]",
     )
-    parser.add_argument(
-        "--no-interactive", help="Don't use interactive", action="store_true"
-    )
     parser.add_argument(
         "-c",
         "--clean",
         action="store_true",
         help="Force clean data, never use cached data",
     )
-    fu_parser = parser.add_argument_group(
```

Comment:
bo idea if this ever did anything :shrug:

Reply:
It did. `no-template-counting` was my personal hell for a while :wink: